docs/setup-aws-lambda.asciidoc

```diff
@@ -1,85 +1,62 @@
 [[aws-lambda]]
-=== Get started with Lambda
+=== Monitoring AWS Lambda Java Functions
+:layer-section-type: with-agent
 
-Getting Elastic APM set up for your lambda functions is easy,
-and there are various ways you can tweak it to fit your needs.
+The Java APM Agent can be used with AWS Lambda to monitor the execution of your AWS Lambda functions.
 
-NOTE: In order to get the full AWS Lambda tracing capabilities, use with APM Server 7.16 or higher. Using
-older versions provides most value, but some relevant metadata fields will not be indexed.
+Follow the <<aws-lambda-java-quick-start>> Guide to get started in minutes and checkout the <<aws-lambda-features-and-caveats>> to learn about the features and limitations of the Java APM Agent on AWS Lambda Functions.
```

Comment:
Some thoughts:

1. I don't think we need to link to the quick start guide when it's the next line down.
2. Will most users be interested in reading the features and caveats prior to starting? It feels weird to send them lower on the page, only to return to the top if they decide to continue.

Reply:
If you want to keep the features and caveats at the bottom, we could include a link in a new "Prerequisites" section, alongside the supported `AWS Lambda runtimes` table (see comment below).

Reply:
The `features and caveats` section isn't really a prerequisite to get started. And I put it after the quick start to focus on a really simple and fast onboarding experience. Maybe we can just remove the link to that section completely in the beginning and rather pointing to it at the end of the quick start section? WDYT?

I'll add my proposal for review to this PR.